src/commands/json.resp.json

```diff
@@ -0,0 +1,26 @@
+{
+    "JSON.RESP": {
+        "summary": "Return the JSON value at the given path in Valkey Serialization Protocol (RESP).",
```

Comment:
I think it's ok to mention Redis Serialization Protocol. 

Valkey Serialization Protocol followed by RESP in bracket is confusing.

Reply:
@madolson any thoughts.